packages/wrapped-keys/src/lib/service-client/client.ts

```diff
@@ -19,17 +19,20 @@ export async function listPrivateKeyMetadata(
 ): Promise<StoredKeyMetadata[]> {
   const { litNetwork, sessionSig } = params;
 
+  const requestId = generateRequestId();
```

Comment:
In the future can we unify the `requestId` being sent to the node with the one sent to the AWS infra? @Ansonhkg

Reply:
Unify how? We can't re-use the exact same ID because there could be more than 1 node request id per call to wrapped keys, or vice-versa.  If you mean the method of creating it, see my [comment here](https://github.com/LIT-Protocol/js-sdk/pull/651#discussion_r1774252142)